### PR TITLE
UIPlugin now conforms with UIObject

### PR DESCRIPTION
**Pull request by leandroalonso**
# Goal

As we're evolving our architecture our first step is to make `UIPlugin` conforming to `UIObject`, having a `View?` and a `render()` method

# How to test

Makes sure all tests pass.

### CODE REVIEW

**Comment by leandroalonso** [comment]
[[Nova UI] Criar MediaControl como plugin no Steve](https://trello.com/c/1kgTo6MV/5-nova-ui-criar-mediacontrol-como-plugin-no-steve)

File: Clappr/Classes/Base/UIObject.swift [hunk 1]
```diff
@@ -0,0 +1,5 @@
+protocol UIObject {
+    var view: UIView? { get set }
```
Comment (CristianDRM) [inline Clappr/Classes/Base/UIObject.swift]: I think that is better to just provide get in order to set the concrete implementation as private(set), what you think?

File: Tests/UIPluginTests.swift [hunk 2]
```diff
@@ -5,45 +5,21 @@ import Nimble
 class UIPluginTests: QuickSpec {
 
     override func spec() {
-        describe("Instantiation") {
+        describe("UIPlugin") {
```
Comment (CristianDRM) [inline Tests/UIPluginTests.swift]: Following the project conventions, the Class name on describe must have the prefix: ``.``

File: Clappr/Classes/Base/UIObject.swift (hunk 1 above)
Comment (leandroalonso) [reply Clappr/Classes/Base/UIObject.swift]: I agree! :)

I will change it, tks. 👍

File: Tests/UIPluginTests.swift (hunk 2 above)
Comment (leandroalonso) [reply Tests/UIPluginTests.swift]: We use `.` for class methods (aka `static`) and `#` for instance methods. The class name is neither of these ones... what do you think?

Comment (CristianDRM) [reply Tests/UIPluginTests.swift]: ;0I ended up confusing the conventions so you can disregard my comment :)

**Pull request merged**